interpreter/cling/lib/Interpreter/DynamicLibraryManagerSymbol.cpp

```diff
@@ -352,6 +352,7 @@ std::string cached_realpath(llvm::StringRef path, llvm::StringRef base_path = ""
       size_t s = result.rfind(sep);
       if (s != llvm::StringRef::npos) result.resize(s);
       if (result.empty()) result = sep;
```

Comment:
My comment seems lost. Can you add new a new line after the if conditions?

Reply:
Addressed in https://github.com/root-project/root/pull/9974